content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
-   :align: center
-   :alt: Score Rule example
+To customize the variables used by predictive lead scoring, go to :menuselection:`CRM -->
+Configuration --> Settings`. Under :guilabel:`Predictive Lead Scoring`, click on the
+:guilabel:`Update Probabilities` button.
 
-Every hour, Odoo automatically assigns scores to new leads according to your configured rules.
+Then, click on the dropdown menu to choose which variables predictive lead scoring will take into
+account.
 
-.. image:: lead_scoring/scoring-section-on-lead.png
+.. image:: lead_scoring/update-probabilities.png
    :align: center
-   :alt: Scoring section on a lead
+   :alt: The Update Probabilities window in the settings for Predictive Lead Scoring.
 
-Assign leads
-============
+Any number of the following variables can be activated:
 
-Odoo can also assign leads to specific teams based on configured rules. To configure assignment
-rules, go to the individual Sales Team's configuration page via :menuselection:`CRM --> Leads -->
-Team Assignment` or :menuselection:`CRM --> Configuration --> Sales Teams`, and apply a specific
-Domain to each team. The Domain can include lead scores.
+- :guilabel:`State`: the geographical state from which the opportunity originates
+- :guilabel:`Country`: the geographical country from which the opportunity originates
+- :guilabel:`Phone Quality`: whether or not a phone number is listed for the opportunity
+- :guilabel:`Email Quality`: whether or not an email address is listed for the opportunity
+- :guilabel:`Source`: the source of an opportunity (e.g. search engine, social media)
+- :guilabel:`Language`: the spoken language specified on the opportunity
+- :guilabel:`Tags`: the tags placed on the opportunity
 
-.. image:: lead_scoring/team-assignation.png
-   :align: center
-   :alt: Team Assignments using domains
+.. note::
+   The variables `Stage` and `Team` are always in effect. `Stage` refers to the CRM pipeline stage
+   that an opportunity is in. `Team` refers to the sales team that is assigned to an opportunity.
+   Predictive lead scoring will *always* take into account these two variables, regardless of which
+   optional variables are selected.
 
-Assign leads to specific salespeople using further refined Domains at :menuselection:`CRM --> Leads
---> Leads Assignment`.
+Next, click on the date next to :guilabel:`Consider leads created as of the:` to select the date
+from which predictive lead scoring will begin its calculations.
 
-.. image:: lead_scoring/lead-assignment-filters.png
-   :align: center
-   :alt: Lead Assignments
+Lastly, click :guilabel:`Confirm` to save changes.
 
-Odoo automatically assigns new leads to teams/salespeople once per day.
+Change the probability manually
+-------------------------------
 
-Evaluate and use the unassigned leads
-=====================================
+The probability of success of an opportunity can be changed manually on the opportunity form. Click
+on the probability number to edit it.
```

Comment:
An opportunity's probability of success can be changed manually on the opportunity form. Click on the probability number to edit it.